pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -86,6 +86,11 @@ func getDBImages(extensionState testhelpers.TestOptions) (prev string, clean str
 		panic("Only use pg12 for upgrade tests")
 	}
 
+	// From Promscale 0.11.0 onwards the minimum extension version is 0.5.0
+	if prevPromscaleVersion != nil && prevPromscaleVersion.GE(semver.MustParse("0.11.0")) {
```

Comment:
I don't understand why can't we use `2.6` with `0.10` ? Is that because `ts2.6` always packs `0.5.0` promscale extension?

Reply:
We want to test the upgrade from the oldest promscale extension version, which is in the `ts2.1` ha docker image.